lib/ClangImporter/SwiftLookupTable.cpp

```diff
@@ -1066,10 +1067,12 @@ namespace {
       uint32_t keyLength = 1;
       if (SwiftLookupTable::contextRequiresName(key.first))
         keyLength += key.second.size();
+      assert(keyLength == static_cast<uint16_t>(keyLength));
 
       // # of entries
       uint32_t dataLength =
         sizeof(uint16_t) + sizeof(uint64_t) * data.size();
+      assert(dataLength == static_cast<uint16_t>(dataLength));
```

Comment:
Shouldn’t this be checking against uint32_t?

Reply:
No, this is a different table, which I didn't change.